tsconfig.json

```diff
@@ -1,13 +1,21 @@
 {
   "compilerOptions": {
     "outDir": "./dist/",
```

Comment:
I don't think there is a problem. this main `tsconfig` file is used for building `dist` - but let me check if the `outDir` has any bearing on that build process.